app/src/main/java/ch/epfl/sweng/runpharaa/user/myProfile/FragmentMyTracks.java

```diff
@@ -71,40 +70,5 @@ public void onError(Exception e) {
                 swipeLayout.setRefreshing(false);
             }
         });
-
-
-        TrackDatabaseManagement.mReadDataOnce(TrackDatabaseManagement.TRACKS_PATH, new Callback<DataSnapshot>() {
```

Comment:
It was my mistake, thank you for noticing

Reply:
Don't worry, was a very quick fix ! :)